stock span

anshmishra010 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Optimized  the code according to Tesseract NeoAlgo standards 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Link :
https://github.com/TesseractCoding/NeoAlgo/blob/master/C-Plus-Plus/stack/Stock_span_problem.cpp

anshmishra010 committed:
stock span

atarax665 committed:
Update Stock_span_problem.cpp

atarax665 commented:
Hey,
Formatting code according to the organization's standard is really a great contribution though I won't be able to add a level tag for all individual PRs. I have added ```Level 0``` for #7230 which will be counted for all your formatting-related PRs.

anshmishra010 commented:
@atarax665  
- I have changed header file .
-  changed the description 
-  formatted the document 
-  and the most important that finding those files which require updation from 2000 + files is not an easy task , so I request you to add the level  as you are only giving 5 marks for it that's it .
-  SO finding the file from SUCH A BIG code base is not easy  . Hope you understand
